src/main/java/core/basesyntax/figure/producers/FigureProducer.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.figure.producers;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.figures.Circle;
+import core.basesyntax.figure.figures.IsoscelesTrapezoid;
+import core.basesyntax.figure.figures.Rectangle;
+import core.basesyntax.figure.figures.RightTriangle;
+import core.basesyntax.figure.figures.Square;
+import java.util.Random;
+
+public class FigureProducer {
+
+    public static Figure getRandomFigure() {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
```
let's not use static methods here

Reply:
we shall not use static methods there!